src/main/java/frc/robot/commands/SetHangArmPosition.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.hang.Hang;
+import frc.robot.subsystems.hang.HangConstants;
+
+public class SetHangArmPosition extends Command {
+  private Hang hang;
+  private double setpoint;
+
+  public SetHangArmPosition(Hang hang, double setpoint) {
+    this.setpoint = setpoint;
+    this.hang = hang;
+
+    addRequirements(hang);
+  }
+
+  @Override
+  public void initialize() {
+    hang.setSetpoint(setpoint);
+  }
+
+  @Override
+  public boolean isFinished() {
+    double i = Math.abs(hang.getPosition());
+
+    return i > (i - HangConstants.TOLERANCE) && i < (i + HangConstants.TOLERANCE);
```

Comment:
This math is incorrect. I suggest you find the absolute difference between the position and the setpoint, then test if it is less than the tolerance.
```suggestion
    return difference < HangConstants.TOLERANCE;
```